webui/react/src/services/decoder.ts

```diff
@@ -3,23 +3,67 @@ import { isNumber } from 'util';
 import dayjs from 'dayjs';
 
 import {
-  decode, ioAgents, ioDetailedUsers, ioDeterminedInfo, ioExperiment, ioExperimentDetails,
-  ioExperiments, ioGenericCommand, ioGenericCommands, ioLog, ioLogin, ioLogs, ioTaskLogs,
-  ioTrialDetails, ioTypeAgents, ioTypeCheckpoint, ioTypeDetailedUsers, ioTypeDeterminedInfo,
-  ioTypeExperiment, ioTypeExperimentConfig, ioTypeExperimentDetails, ioTypeExperiments,
-  ioTypeGenericCommand, ioTypeGenericCommands, ioTypeLog, ioTypeLogin, ioTypeLogs, ioTypeMetric,
-  ioTypeStep, ioTypeTaskLogs, ioTypeTrial, ioTypeTrialDetails, ioTypeValidationMetrics,
+  decode,
+  ioDetailedUsers,
+  ioDeterminedInfo,
+  ioExperiment,
+  ioExperimentDetails,
+  ioExperiments,
+  ioGenericCommand,
+  ioGenericCommands,
+  ioLog,
+  ioLogin,
+  ioLogs,
+  ioTaskLogs,
+  ioTrialDetails,
+  ioTypeCheckpoint,
+  ioTypeDetailedUsers,
+  ioTypeDeterminedInfo,
+  ioTypeExperiment,
+  ioTypeExperimentConfig,
+  ioTypeExperimentDetails,
+  ioTypeExperiments,
+  ioTypeGenericCommand,
+  ioTypeGenericCommands,
+  ioTypeLog,
+  ioTypeLogin,
+  ioTypeLogs,
+  ioTypeMetric,
+  ioTypeStep,
+  ioTypeTaskLogs,
+  ioTypeTrial,
+  ioTypeTrialDetails,
+  ioTypeValidationMetrics,
```

Comment:
**suggestion:** might have to try this out to make sure no funny circular dependency happens, but with this many individual items being imported, it might just be better to do this:
```
import * as ioTypes from 'ioTypes';
import * as types from 'types';
```

And reference them in the decoder like `ioTypes.ioTypeTrial` and `types.Agent`.

Reply:
Done.